### PR TITLE
Redirect not working for landing pages

### PR DESCRIPTION
**Pull request by dadarya0**
**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? |  Y
| New feature? | 
| Automated tests included? | Y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
We were using redirection type drop down values from config file. May be due to wrong value in config or any other issue landing pages redirection url was not working.
Now we have added redirection type list hard coded.

#### Steps to test this PR:
1. Create a landing page (leave unpublished)
2. Select redirect type 301 or 302 and add Redirect URL
3. Visit the landing page URL in incognito mode/different browser
4. Should be redirected to the redirect URL

#### Other areas of Mautic that may be affected by the change:
1. 
2. 

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.

### CODE REVIEW

**Review by kuzmany — DISMISSED** [review]
Works for me. 
Nice job 👍

**Comment by RCheesley** [comment]
@dadarya0 this one needs to be rebased to 4.x please so that we can consider it for merging in the next release. If it's easier you could also create a new PR and just reference it back to this one. Sorry for the inconvenience!

**Comment by RCheesley** [comment]
This may not need more automated tests but it seems the code coverage check is failing, so I would appreciate a review by a developer to determine if we should be expecting a reduction.

File: app/bundles/PageBundle/Form/Type/RedirectListType.php [hunk 1]
```diff
@@ -12,15 +11,19 @@
  */
 class RedirectListType extends AbstractType
 {
-    public function __construct(
-        private CoreParametersHelper $coreParametersHelper
-    ) {
+    public function __construct()
```
Comment (escopecz) [inline app/bundles/PageBundle/Form/Type/RedirectListType.php]: Can we remove this construct if it's not doing anything?

**Review by escopecz — CHANGES_REQUESTED** [review]

File: app/bundles/PageBundle/Form/Type/RedirectListType.php (hunk 1 above)
Comment (rohitpavaskar) [reply app/bundles/PageBundle/Form/Type/RedirectListType.php]: Removed. Thanks!

**Review by escopecz — APPROVED** [review]
The code looks great, the redirect works even if the page is unpublished 👍

**Pull request merged**